pgxn/neon/walproposer_pg.c

```diff
@@ -68,6 +68,8 @@ int			safekeeper_proto_version = 3;
 char	   *safekeeper_conninfo_options = "";
 /* BEGIN_HADRON */
 int         databricks_max_wal_mb_per_second = -1;
+static uint32 kShouldLimit = 1;
```

Comment:
Why this variables are introduced?
Why not just bool?

Reply:
See https://github.com/databricks-eng/hadron/pull/1365. Went ahead and just cleaned up. Can cherry-pick it into this PR later.